examples/non-fungible/src/lib.rs

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::fmt::{Display, Formatter};
+
+use async_graphql::{Request, Response, SimpleObject};
+use fungible::Account;
+use linera_sdk::{
+    base::{AccountOwner, ApplicationId, ChainId, ContractAbi, ServiceAbi},
+    graphql::GraphQLMutationRoot,
+};
+use serde::{Deserialize, Serialize};
+
+pub type TokenId = String;
+
+pub struct NonFungibleTokenAbi;
+
+impl ContractAbi for NonFungibleTokenAbi {
+    type InitializationArgument = ();
+    type Parameters = ();
+    type ApplicationCall = ApplicationCall;
```

Comment:
Minor: I think you can use `Operation` here, and remove the `ApplicationCall` type entirely.

```suggestion
    type ApplicationCall = Operation;
```

(I'll also do this for all other applications in an upcoming PR for #1752)